src/Datadog.Trace/TracingProcessManager.cs

```diff
@@ -182,12 +237,15 @@ private static Task StartProcessWithKeepAlive(ProcessMetadata metadata)
                         metadata.IsBeingManaged = false;
                     }
                 },
-                _cancellationTokenSource.Token);
+                TaskCreationOptions.LongRunning);
```

Comment:
Given the lambda uses `await`, doesn't that negate the utility of `TaskCreationOptions.LongRunning`, as the thread will be returned anyway?